pkg/plugins/app_plugin.go

```diff
@@ -67,6 +72,15 @@ func (app *AppPlugin) Load(decoder *json.Decoder, base *PluginBase, backendPlugi
 		return err
 	}
 
+	if app.Backend {
+		cmd := ComposePluginStartCommand(app.Executable)
+		fullpath := path.Join(app.PluginDir, cmd)
+		factory := grpcplugin.NewBackendPlugin(app.Id, fullpath, grpcplugin.PluginStartFuncs{})
+		if err := backendPluginManager.Register(app.Id, factory); err != nil {
+			return errutil.Wrapf(err, "Failed to register backend plugin")
```

Comment:
Go errors should not be capitalized:
```suggestion
			return errutil.Wrapf(err, "failed to register backend plugin")
```